website/eslint.config.mjs

```diff
@@ -0,0 +1,44 @@
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+import react from 'eslint-plugin-react';
+import reactHooks from 'eslint-plugin-react-hooks';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
+  {
+    ignores: [
+      '.docusaurus/**',
+      'build/**',
+      'node_modules/**',
+      'scripts/**',
+      'static/**',
+      'docusaurus.config.ts',
```

Comment:
Why do we need ignore this?

Reply:
fix: [9594e8f](https://github.com/enactic/openarm/pull/237/commits/9594e8fb02b709d6fb6f0f1cf1c60f7090755736) Yes, we can. I added it's because there was ESLint errors during implementation steps.